index.md

```diff
@@ -10,7 +10,7 @@ language: "FIXME"     # lowercase two-letter ISO language code such as "fr" (see
 latitude: "45"        # decimal latitude of workshop venue (use https://www.latlong.net/)
 longitude: "-1"       # decimal longitude of the workshop venue (use https://www.latlong.net)
 humandate: "FIXME"    # human-readable dates for the workshop (e.g., "Feb 17-18, 2020")
-humantime: "FIXME"    # human-readable times for the workshop (e.g., "9:00 am - 4:30 pm")
+humantime: "FIXME"    # human-readable times for the workshop e.g., "9:00 am - 4:30 pm CEST<br>(7:00 am - 2:30 pm UTC)" or "9:00 am - 4:30 pm"
```

Comment:
```suggestion
humantime: "FIXME"    # human-readable times for the workshop e.g., "9:00 am - 4:30 pm CEST (7:00 am - 2:30 pm UTC)"
```

I really like this suggestion! Two suggested changes: IMO the use of the <br> tag will make the template less accessible to those without much experience of HTML. And I am in favour of removing the second example.

Reply:
Looks great, thank you @tobyhodges